### ISSUE TITLE
Error: Duplicate element found

### ISSUE THREAD

**Issue by eBaeza** [IckleChris/jquery-date-dropdowns#9]
I am trying to setup two date-dropdowns but i get this error "Uncaught Error: Duplicate element found"
I check the attribute "name" for both and those are different

```
<input type="hidden" name="start_date_mob" id="start_date_mob" class="date_dropdown">

<input type="hidden" name="finish_date_mob" id="finish_date_mob" class="date_dropdown">
```

```
$('.date_dropdown').dateDropdowns
        defaultDateFormat: 'dd/mm/yyyy'
        submitFormat: 'dd/mm/yyyy'
        defaultDate: day + '/' + month + '/' + year
        minYear: year
        maxYear: year
```

**Comment by IckleChris** [comment]
This is related to issue #8 as existing `name` attributes are being overwritten by the plugin's default unless it is provided in the configuration. As a temporary fix you can instantiate each element separately and provide the `submitFieldName` value. E.g.

```
$('#start_date_mob').dateDropdowns({
    submitFieldName: 'start_date_mob',
    // ...
});
```

A fix for this will be available shortly.

**Comment by eBaeza** [comment]
Thanks!!! i can solve my incident.

**Comment by josenova** [comment]
Hi Chris. Im trying to use this on ApplicantStacks forms. The inputs are already created and have a name like "qtn_102361_3213974". When I try to use the `submitFieldName` property to give the input its name back I get the Duplicate Element Found error.

Any suggestions?

**Comment by IckleChris** [comment]
This has been fixed (see #8)
